core/src/main/java/tc/oc/pgm/api/player/Username.java

```diff
@@ -20,7 +20,7 @@
    * @return The username.
    */
   @Nullable
-  String getName();
+  String getUsername();
```

Comment:
Why change this and `Contributor`?

Reply:
In my latest commit I resolved the issue related to `Contributor`. Though I left Username alone and would like some guidance on it. 

For `Contributor` I made use of the `getName()` method and included a `toLegacy()` that covers a few places that using a component would prove challenging (e.g. /maps -a flag to match author names), however I was unsure if this was the correct way to go for `Username`. As there are a few places in the code where it’s acceptable for the username to be null, and currently the `Username#getName()` will always return a component. So perhaps it makes sense to keep `Username#getUsername` for when it’s necessary to check if the name is null or not.

 Let me know how you’d like this to be handled. Thanks!